Paintroid/src/main/java/org/catrobat/paintroid/tools/implementation/DefaultToolReference.kt

```diff
@@ -16,22 +16,17 @@
  * You should have received a copy of the GNU Affero General Public License
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
+package org.catrobat.paintroid.tools.implementation
 
-package org.catrobat.paintroid.tools.implementation;
+import org.catrobat.paintroid.tools.Tool
+import org.catrobat.paintroid.tools.ToolReference
 
-import org.catrobat.paintroid.tools.Tool;
-import org.catrobat.paintroid.tools.ToolReference;
+class DefaultToolReference : ToolReference {
+    private var tool: Tool? = null
```

Comment:
Can this be done like:
```suggestion
    override var tool: Tool? = null
```
?

Reply:
I was also thinking a similar change, I'll try and implement a similar idea :)